packages/blade/src/components/Chip/_decisions/decisions.md

```diff
@@ -0,0 +1,152 @@
+| **intent**              | `'positive' \| 'negative' \| 'neutral'` | `'neutral'` | Sets the ChipGroups's visual variant, it will propagate down to all the Chips                                                                      | ❌       |
+| **value (html native)** | `string`                                | `undefined` | The value of the input field in Chip, [useful in form submissions](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#value) | ❌       |
+
+## `ChipGroup` API
+
+| Prop                   | Type                                                             | Default     | Description                                                                                                                    | Required |
+| ---------------------- | ---------------------------------------------------------------- | ----------- | ------------------------------------------------------------------------------------------------------------------------------ | -------- |
+| **accessibilityLabel** | `string`                                                         | `undefined` | Accessibility label for the ChipGroup                                                                                          | ✅       |
+| **children**           | `React.ReactNode`                                                | `undefined` | Accepts multiple `Chip` components                                                                                             | ✅       |
+| **defaultValue**       | `string \|string[]`                                              | `undefined` | Sets the initial value of the Chip group                                                                                       | ❌       |
+| **isDisabled**         | `boolean`                                                        | `false`     | Disables or enables ChipGroup, it will propagate down to all the Chips                                                         | ❌       |
+| **name**               | `string`                                                         | `undefined` | The name of the chip group, [useful in form submissions](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#name) | ❌       |
+| **onChange**           | `({ name, values }: { name: string; values: string[] }) => void` | `undefined` | Calls a function on any state change within the ChipGroup                                                                      | ❌       |
+| **selectionType**      | `'single' \| 'multiple'`                                         | `'single'`  | Defines the selection behavior within the ChipGroup component, simulating radio/checkbox like behavior within the group        | ❌       |
+| **size**               | `'xsmall' \| 'small' \| 'medium' \| 'large'`                     | `'small'`   | Specifies the size of the rendered Chips withing the ChipGroup                                                                 | ❌       |
```

Comment:
@anshulsahni This time we've considered a large size as well that can work for landing pages. Also, got it reviewed by advocates including Acquisition team.

Being said that, it might not solve the use-case for assisted discovery because that is still a custom component and cannot be made generic.

Reply:
@saurav12 One doubt, the large variant is present in other components, I don't see an variant larger than that, any thing different with the larger variant of Chip component vs. other components?

Reply:
@anshulsahni We have aligned the larger variant of the `Chip` component to the `Button` component that we have in the DS. We've got feedback that this can be used as an actionable entity on Desktop devices (for ex. It can be on the onboarding screen, assisted discovery, and partnership onboarding) and hence it should be similar to actionable components (i.e. buttons).